docs/New-Maintainer-Checklist.md

```diff
@@ -50,15 +50,27 @@ Thanks for all your work so far!
+- Invite them to the [**@Homebrew/maintainers** team](https://github.com/orgs/Homebrew/teams/maintainers) (or any relevant [subteams](https://github.com/orgs/Homebrew/teams/maintainers/teams)) to give them write access to relevant repositories (but don't make them owners). They will need to enable [GitHub's Two Factor Authentication](https://help.github.com/articles/about-two-factor-authentication/).
 - Ask them to sign in to [Bintray](https://bintray.com) using their GitHub account and they should auto-sync to [Bintray's Homebrew organisation](https://bintray.com/homebrew/organization/edit/members) as a member so they can publish new bottles
 - Add them to the [Jenkins' GitHub Authorization Settings admin user names](https://jenkins.brew.sh/configureSecurity/) so they can adjust settings and restart jobs
 - Add them to the [Jenkins' GitHub Pull Request Builder admin list](https://jenkins.brew.sh/configure) to enable `@BrewTestBot test this please` for them
 - Invite them to the [`homebrew-maintainers` private maintainers mailing list](https://lists.sfconservancy.org/mailman/admin/homebrew-maintainers/members/add)
 - Invite them to the [`machomebrew` private maintainers Slack](https://machomebrew.slack.com/admin/invites) (and ensure they've read the [communication guidelines](Maintainer-Guidelines.md#communication))
-- Invite them to the [`homebrew` private maintainers 1Password](https://homebrew.1password.com/signin)
+- Add them to [Homebrew's README](https://github.com/Homebrew/brew/edit/master/README.md)
+
+If they are also interested in doing system administration work:
+
+- Invite them to the [`homebrew-ops` private operations mailing list](https://lists.sfconservancy.org/mailman/admin/homebrew-ops/members/add)
+- Invite them to the [`homebrew` private 1Password](https://my.1password.com/people)
+
+If they want to consume raw anonymous aggregate analytics data (rather than use `brew formula-analytics`):
+
 - Invite them to [Google Analytics](https://analytics.google.com/analytics/web/?authuser=1#management/Settings/a76679469w115400090p120682403/%3Fm.page%3DAccountUsers/)
 
-After a month-long trial period with no problems make them [owners on the Homebrew GitHub organisation](https://github.com/orgs/Homebrew/people) and add them to [Homebrew's README](https://github.com/Homebrew/brew/edit/master/README.md). If there are problems, ask them to step down as a maintainer and revoke their access to the above.
+Once they have been active maintainers for at least a year:
+
+- Invite them to join Homebrew's Software Freedom Conservancy Project Leadership Committee. If they accept, email their name, email and employer to homebrew@sfconservancy.org and make them [owners on the Homebrew GitHub organisation](https://github.com/orgs/Homebrew/people).
```

Comment:
Make that `[Software Freedom Conservancy](https://sfconservancy.org) Project Leadership Committee`, not only for the link’s sake but also to clarify where the entity name ends and the committee name begins.

Reply:
Good call, thanks @claui!